### ISSUE TITLE
[Bug]: [PZ] Server doesn't seem to start in LGSM Containers

### ISSUE THREAD

**Issue by ocelik94** [GameServerManagers/LinuxGSM#4573]
### User story

As a user I want to run a pzserver with default settings and be able to connect to it

### Game

Project Zomboid

### Linux distro

Ubuntu 22.04, Other

### Command

command: start

### Further information

Used docker-compose.yml:
```
services:
  server:
    image: gameservermanagers/gameserver:pz
    restart: unless-stopped
    ports:
      - "16261:16261"
      - "16262:16262"
    volumes:
      - ./data/pzserver:/data
```

I am not being able to connect to the server at all. If I am correct the start-script of the pz server doesnt listen to the ports..

### Relevant log output

```shell
server-1  | ==> /data/log/script/pzserver-script.log <==
server-1  | May 08 09:48:59.737 pzserver: START: PASS: Started LinuxGSM
server-1  | tail: cannot open '/data/log/console/*script.log' for reading: No such file or directory
server-1  | tail: cannot open '/data/log/script/*console.log' for reading: No such file or directory
server-1  | May 08 09:48:59.841 pzserver: START: PASS: Started LinuxGSM
server-1  | May 08 09:48:59.848 pzserver: START: INFO: LinuxGSM version: v24.2.1
server-1  | May 08 09:48:59.849 pzserver: START: PASS: core_exit.sh exiting with code: 0
server-1  | May 08 09:49:05.151 pzserver: DETAILS: PASS: Using cached IP as public IP address
server-1  | May 08 09:49:05.475 pzserver: DETAILS: PASS: Using cached IP as public IP address
server-1  | May 08 09:49:18.208 pzserver: DETAILS: INFO: LinuxGSM version: v24.2.1
server-1  | May 08 09:49:18.209 pzserver: DETAILS: PASS: core_exit.sh exiting with code: 0
server-1  | May 08 09:49:43.131 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:49:43.212 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:49:44.757 pzserver: MONITOR: INFO: Checking session: CHECKING
server-1  | May 08 09:49:44.968 pzserver: MONITOR: PASS: Checking session: OK
server-1  | May 08 09:49:46.473 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 1 : QUERYING
server-1  | May 08 09:49:47.580 pzserver: MONITOR: INFO: Querying port: gamedig: 0.0.0.0:16261 : 1 : DELAY
server-1  | May 08 09:49:47.581 pzserver: MONITOR: INFO: Query bypassed: pzserver started less than 1 minutes ago
server-1  | May 08 09:49:47.585 pzserver: MONITOR: INFO: Server started: Wed May  8 09:48:59 AM UTC 2024
server-1  | May 08 09:49:47.587 pzserver: MONITOR: INFO: Current time: Wed May  8 09:49:47 AM UTC 2024
server-1  | May 08 09:49:47.592 pzserver: MONITOR: INFO: LinuxGSM version: v24.2.1
server-1  | May 08 09:49:47.593 pzserver: MONITOR: PASS: core_exit.sh exiting with code: 0
server-1  | May 08 09:50:48.403 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:50:48.481 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:50:50.028 pzserver: MONITOR: INFO: Checking session: CHECKING
server-1  | May 08 09:50:50.239 pzserver: MONITOR: PASS: Checking session: OK
server-1  | May 08 09:50:51.745 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 1 : QUERYING
server-1  | May 08 09:50:56.171 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 1 : FAIL
server-1  | May 08 09:50:57.676 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 1 : QUERYING
server-1  | May 08 09:51:02.103 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 1 : FAIL
server-1  | May 08 09:51:03.608 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 1 : QUERYING
server-1  | May 08 09:51:08.032 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 1 : FAIL
server-1  | May 08 09:51:26.079 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 2 : QUERYING
server-1  | May 08 09:51:30.508 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 2 : FAIL
server-1  | May 08 09:51:32.012 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 2 : QUERYING
server-1  | May 08 09:51:36.439 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 2 : FAIL
server-1  | May 08 09:51:37.944 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 2 : QUERYING
server-1  | May 08 09:51:42.374 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 2 : FAIL
server-1  | May 08 09:52:00.422 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 3 : QUERYING
server-1  | May 08 09:52:04.852 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 3 : FAIL
server-1  | May 08 09:52:06.357 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 3 : QUERYING
server-1  | May 08 09:52:10.785 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 3 : FAIL
server-1  | May 08 09:52:12.289 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 3 : QUERYING
server-1  | May 08 09:52:16.717 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 3 : FAIL
server-1  | May 08 09:52:34.765 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 4 : QUERYING
server-1  | May 08 09:52:39.192 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 4 : FAIL
server-1  | May 08 09:52:40.697 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 4 : QUERYING
server-1  | May 08 09:52:45.125 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 4 : FAIL
server-1  | May 08 09:52:46.629 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 4 : QUERYING
server-1  | May 08 09:52:48.444 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:52:48.522 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:52:50.068 pzserver: MONITOR: INFO: Checking session: CHECKING
server-1  | May 08 09:52:50.280 pzserver: MONITOR: PASS: Checking session: OK
server-1  | May 08 09:52:51.057 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 4 : FAIL
server-1  | May 08 09:52:51.785 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 1 : QUERYING
server-1  | May 08 09:52:56.210 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 1 : FAIL
server-1  | May 08 09:52:57.714 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 1 : QUERYING
server-1  | May 08 09:53:02.140 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 1 : FAIL
server-1  | May 08 09:53:03.645 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 1 : QUERYING
server-1  | May 08 09:53:08.074 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 1 : FAIL
server-1  | May 08 09:53:09.104 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 5 : QUERYING
server-1  | May 08 09:53:13.535 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 5 : FAIL
server-1  | May 08 09:53:13.740 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 5 : FAIL
server-1  | May 08 09:53:13.744 pzserver: MONITOR: INFO: Sending alert: Unable to query pzserver. Game server has been restarted
server-1  | May 08 09:53:13.975 pzserver: MONITOR: PASS: Using cached IP as public IP address
server-1  | May 08 09:53:14.672 pzserver: RESTART: PASS: Using cached IP as public IP address
server-1  | May 08 09:53:15.312 pzserver: STOP: PASS: Using cached IP as public IP address
server-1  | May 08 09:53:15.887 pzserver: STOP: PASS: Using cached IP as public IP address
server-1  | May 08 09:53:16.424 pzserver: STOP: INFO: Graceful: sending "quit"
server-1  | 
server-1  | ==> /data/log/console/pzserver-console.log <==
server-1  | 
server-1  | quit
server-1  | LOG  : General     , 1715161996426> 49,102,591> Enter new administrator password: 
server-1  | LOG  : General     , 1715161996426> 49,102,592> Confirm the password: 
server-1  | 
server-1  | ==> /data/log/script/pzserver-script.log <==
server-1  | May 08 09:53:26.121 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 2 : QUERYING
server-1  | May 08 09:53:30.548 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 2 : FAIL
server-1  | May 08 09:53:32.052 pzserver: MONITOR: INFO: Querying port: gamedig: 172.20.0.2:16261 : 2 : QUERYING
server-1  | May 08 09:53:36.478 pzserver: MONITOR: WARN: Querying port: gamedig: 172.20.0.2:16261 : 2 : FAIL
server-1  | May 08 09:53:37.982 pzserver: MONITOR: INFO: Querying port: gamedig: 162.55.224.48:16261 : 2 : QUERYING
server-1  | May 08 09:53:42.414 pzserver: MONITOR: WARN: Querying port: gamedig: 162.55.224.48:16261 : 2 : FAIL
server-1  | May 08 09:54:00.460 pzserver: MONITOR: INFO: Querying port: gamedig: 127.0.0.1:16261 : 3 : QUERYING
server-1  | May 08 09:54:01.907 pzserver: STOP: ERROR: Graceful: sending "quit": FAIL
server-1  | May 08 09:54:02.417 pzserver: STOP: INFO: tmux kill-session: pzserver: LinuxGSM
server-1  | May 08 09:54:03.531 pzserver: STOP: PASS: Stopped LinuxGSM
server-1  | May 08 09:54:04.885 pzserver: START: PASS: Checking ubuntu-22.04.csv
server-1  | May 08 09:54:04.892 pzserver: MONITOR: WARN: Querying port: gamedig: 127.0.0.1:16261 : 3 : FAIL
server-1  | May 08 09:54:05.138 pzserver: START: PASS: Using cached IP as public IP address
server-1  | May 08 09:54:05.190 pzserver: START: INFO: Using anonymous Steam login
server-1  | May 08 09:54:05.500 pzserver: START: PASS: Using cached IP as public IP address
server-1  | May 08 09:54:06.050 pzserver: START: INFO: Rotating log files
```

### Steps to reproduce

basically just run the vanilla server in docker

**Comment by ocelik94** [comment]
I think I found the failure. Could it be that the administrator password prompt is new?

```
server-1  | LOG  : General     , 1715161996426> 49,102,591> Enter new administrator password: 
server-1  | LOG  : General     , 1715161996426> 49,102,592> Confirm the password: 
```

I tried setting the password manually with ./pzserver console and after that the server seems to start correctly

**Comment by albertovecina** [comment]
Hi @ocelik94 I'm new using docker (and project zomboid). Could you explain how to set the password step by step? Thanks!

**Comment by ocelik94** [comment]
Hey @albertovecina,
there are probably different ways to do that.

You could attack to the container with `docker attach` or open a bash inside the container with `docker compose exec -it --user linuxgsm server bash` and run the server manually once with interactive shell `./pzserver start`

After setting the password initially it will work as long you use the same save

**Comment by irobot73** [comment]
It's not 100% obvious, but you can update the /data/config-lgsm/pzserver/_defautls.cfg

```
## Predefined Parameters | https://docs.linuxgsm.com/configuration/start-parameters
ip="0.0.0.0"
adminpassword="CHANGE_ME"
```

Though, better to copy the relevant sections into the .../common.cfg instead & modify as you wish.

```
## Predefined Parameters | https://docs.linuxgsm.com/configuration/start-parameters
adminusername="pz_admin"
adminpassword="{password_here}"
selfname=pzserver

## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
#
#
# First run (will create & sync account created for admin, any default 'admin' should be disabled in DB)
#startparameters="-servername ${selfname} -adminusername \"${adminusername}\" -adminpassword \"${adminpassword}\""
#
# Standard LGSM
startparameters="-servername ${selfname} -adminusername \"${adminusername}\""
```

Here I've run once to setup the ADMIN w/ the PW I choose, then reverse the '#' & everything after uses the 'Standard LGSM' (else, IIRC, it'll keep asking to create the default 'ADMIN' account)...which *I* don't want.
